lib/CMakeLists.txt

```diff
@@ -0,0 +1,158 @@
+include(ExternalProject)
+
+# External project versions
+set(LIQ_VERSION "2.12.2")
+set(EXIF_VERSION "0.6.22")
+string(REPLACE "." "_" EXIF_VERSION_TAG ${EXIF_VERSION})
+set(JPG_VERSION "2.0.2")
+set(PNG_VERSION "1.6.34")
+set(GIF_VERSION "5.1.4")
+set(WEBP_VERSION "1.0.2")
+set(LCMS2_VERSION "2.9")
+set(VIPS_VERSION "8.9.0")
+set(VIPS_VERSION "${VIPS_VERSION} PARENT_SCOPE)
```

Comment:
Missing double-quote.

Reply:
Yes, I didn't push the last version. I have to find a way to retrieve JVIPS_VERSION into the build.sh without lib/variables.sh